### ISSUE TITLE
snap on Linux Mint 18.1 Qt and Gtk error

### ISSUE THREAD

**Issue by balazsdukai** [cnr-isti-vclab/meshlab#160]
I installed Meshlab with the snap package and I'm able to run it without noticeable issues (so far), however it still gives the error message below. I have both  `libatk-adaptor` and  `libgail-common` installed. My system is:

![screenshot from 2017-06-01 12-13-50](https://cloud.githubusercontent.com/assets/7190603/26675561/bbbb0aee-46c4-11e7-8ac7-20a561c4f8b0.png)

The error message:

```
Gtk-Message: Failed to load module "gail"
Gtk-Message: Failed to load module "atk-bridge"
Qt: Session management error: None of the authentication protocols specified are supported
QNetworkManagerInterface::QNetworkManagerInterface(QObject*) propsReply "An AppArmor policy prevents this sender from sending this message to this recipient; type=\"method_call\", sender=\":1.238\" (uid=1000 pid=32530 comm=\"meshlab \") interface=\"org.freedesktop.DBus.Properties\" member=\"GetAll\" error name=\"(unset)\" requested_reply=\"0\" destination=\"org.freedesktop.NetworkManager\" (uid=0 pid=26520 comm=\"/usr/sbin/NetworkManager --no-daemon \")"
QNetworkManagerInterface::QNetworkManagerInterface(QObject*) nmReply "An AppArmor policy prevents this sender from sending this message to this recipient; type=\"method_call\", sender=\":1.238\" (uid=1000 pid=32530 comm=\"meshlab \") interface=\"org.freedesktop.NetworkManager\" member=\"GetDevices\" error name=\"(unset)\" requested_reply=\"0\" destination=\"org.freedesktop.NetworkManager\" (uid=0 pid=26520 comm=\"/usr/sbin/NetworkManager --no-daemon \")"
"Object path cannot be empty"
Current Plugins Dir is: /snap/meshlab/4/bin/plugins 
"Cannot load library /snap/meshlab/4/bin/plugins/libfilter_ssynth.so: (/snap/meshlab/4/bin/plugins/libfilter_ssynth.so: undefined symbol: _ZTIN14StructureSynth5Model9Rendering16TemplateRendererE)"
```

**Comment by lijgame** [comment]
I'm experiencing the same issue on Linux Mint 18.2.
Any solutions?

**Comment by neycyanshi** [comment]
still not solved on Linux Ubuntu 16.04 LTS. Installed from mardy's meshlab-mardy snap. Any one can help?

**Comment by alemuntoni** [comment]
This issue should be solved in the latest version (2020.06). Feel free to reopen if the problem persists.
